src/reducers/index.js

```diff
@@ -1,8 +1,10 @@
 import { combineReducers } from 'redux';
 import authReducer from './authReducer';
 import signupSignin from './signupSignupReducer';
+import resetPasswordReducer from './resetReducer';
```

Comment:
The import name does not match the name of the file.

Read the `Component Naming` here https://github.com/airbnb/javascript/tree/master/react to learn more

Reply:
noted